corehq/apps/campaign/tests/test_models.py

```diff
@@ -46,9 +52,10 @@ def dashboard_maps(dashboard):
     ]
 
 
-@pytest.fixture
-def dashboard_reports(dashboard):
-    return [
+@fixture
+def dashboard_reports():
+    dashboard = dashboard_fixture()
+    yield [
         DashboardReport.objects.create(
```

Comment:
Similar to the previous comment, I think this yielded list is not used, although the things constructed to create it are probably important.